### DISCUSSION TITLE
Proposal About 8.0/8.1

### DISCUSSION THREAD

**Discussion by GrahamCampbell** [shivammathur/setup-php#284] [Ideas / RFC] [answered]
The `master` branch (of php/php-src) will soon be becoming PHP 8.1, I think a couple of days before PHP 8.0.0RC1 is released. When that happens, should we make `8.0` point to the latest RC release (and then, later, to stable releases), rather than nightlies, and then `8.1` point to nightly master builds?

**Comment by shivammathur** [comment, marked as answer]
@GrahamCampbell  Yes, I was thinking in `shivammathur/php-builder` when `php-src` branches off, I will change `master` to the new branch in `install_package_from_github` call. 

When `RC/stable` packages from `ppa:ondrej/php` are available for `8.0`. I will revert it to `master` and add `8.1` in `setup-php`.

Let me know your thoughts, if this would be ok.

**Comment by GrahamCampbell** [comment]
Sounds fine. I think the RC release will happen 2 days after the branch change. Not sure when ondrej/php will be ready.
